packages/admin/src/components/LoginForm/index.tsx

```diff
@@ -64,24 +62,23 @@ export default function LoginForm() {
           />
         </div>
         <div
-          className={cx(style.box, style.pwBox, {
-            [style.focus]: isFocusPw,
+          className={cx($.pwBox, {
+            [$.focus]: isFocusPw,
           })}
         >
           <button
-            className={style.lockBox}
+            className={$.lockBox}
```

Comment:
```suggestion
            className={$['lock-box']}
```
저번 회의때 정한대로 케밥케이스로 부탁드립니다~